### PR TITLE
fix: Fixes cc bcc being sent when mail head is empty  

### PR DESCRIPTION
**Pull request by nithindavid**
Fixes: #3503

### CODE REVIEW

File: app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue [hunk 1]
```diff
@@ -309,6 +310,9 @@ export default {
         this.showMentions = false;
       }
     },
+    ccEmails(newVal) {
```
Comment (fayazara) [inline app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue:313]: Can remove this block.

**Review by fayazara — CHANGES_REQUESTED** [review]
Looks good. Minor correction

File: app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue [hunk 2]
```diff
@@ -280,7 +281,7 @@ export default {
       return !this.message.trim().replace(/\n/g, '').length;
     },
     showReplyHead() {
-      return !this.isOnPrivateNote && this.isAnEmailChannel;
+      return true || (!this.isOnPrivateNote && this.isAnEmailChannel);
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue:284]: Revert this change since reply email head only for email inboxes

Comment (nithindavid) [inline app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue:284]:
```suggestion
      return !this.isOnPrivateNote && this.isAnEmailChannel;
```
